src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -12,12 +13,12 @@ namespace LaunchDarkly.EventSource
     /// </summary>
     public class EventSource : IEventSource, IDisposable
     {
-
         #region Private Fields
 
         private readonly Configuration _configuration;
         private readonly ILog _logger;
 
+        private HttpClient _httpClient;
```

Comment:
This is the biggest functional change, especially in the context of the Xamarin SDK. Previously, we were creating a new `HttpClient` for every connection attempt. In server-side .NET that may not be a big deal, but in a mobile environment we do a lot more reconnecting, and since the client properties are always the same I decided to just use one of them and move it up here.